spec/features/item_show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe "Item show" do
+  let(:item) { create(:item) }
+
+  it "displays item title" do
+    visit item_path(item)
+    within('.item-title') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.